### DISCUSSION TITLE
Minecraft Server 1.17 Java Edition (mcserver)

### DISCUSSION THREAD

**Discussion by vfg2151** [GameServerManagers/LinuxGSM#3480] [General]
Now that Minecraft 1.17 is out it won't just be enough to update the server. Everyone, new and existing servers must also update their Java version to 16:

`sudo apt install openjdk-16-jre-headless
`
When this is done your Minecraft server will start and run 1.17.

**Comment by dgibbs64** [comment]
Thanks. I will be addressing this in the next hotfix
